cache/src/main/java/com/uber/buckcache/resources/buckcache/BuckCacheResource.java

```diff
@@ -111,7 +113,8 @@ public Response getCacheArtifact(@PathParam("key") String key) throws Exception
   @Path("key")
   @Consumes(MediaType.APPLICATION_OCTET_STREAM)
   public Response addArtifactToCache(PutCacheEntry putCacheEntry,
-                                     @HeaderParam(X_CACHE_EXPIRY_SECONDS) String cacheExpirySeconds) throws Exception {
+                                     @HeaderParam(X_CACHE_EXPIRY_SECONDS) String cacheExpirySeconds,
+                                     @Auth @HeaderParam(HttpHeaders.AUTHORIZATION) String auth) throws Exception {
```

Comment:
its used by `HttpHeaderAuthFilter`, but I change the code to call `@PermitAll` to activate the authentication